camelot/types/__init__.py

```diff
@@ -330,7 +330,7 @@ def result_processor(self, dialect, coltype=None):
         def processor(value):
             if value:
                 value = impl_processor(value)
-                return StoredFile(self.storage, PurePosixPath(value))
+                return StoredFile(self.storage, PurePath(value))
```

Comment:
gaat dit werken ?

want value is eigenlijk een posix path ?

Reply:
Ja, ik heb het eens kort getest. Het voordeel is denk ik dat in windows forward slashes ook gebruik worden. 
```
PureWindowsPath('/test/test2/testfile.pdf')
Out[3]: PureWindowsPath('/test/test2/testfile.pdf')
p = PureWindowsPath('/test/test2/testfile.pdf')
p.as_posix()
Out[5]: '/test/test2/testfile.pdf'
str(p)
Out[6]: '\\test\\test2\\testfile.pdf'
p.parent
Out[7]: PureWindowsPath('/test/test2')
```

Reply:
ok, heb nog wat extra testen gedaan, ziet er idd ok uit

`
import pathlib
wp = pathlib.PureWindowsPath(r'C:\Program Files\Bazaar\crash.exe')
posix_str = wp.as_posix()
posix_str
'C:/Program Files/Bazaar/crash.exe'
type(posix_str)
<class 'str'>
back_to_win = pathlib.PureWindowsPath(posix_str)
back_to_win
PureWindowsPath('C:/Program Files/Bazaar/crash.exe')
wp == back_to_win
True
back_to_win.drive()
Traceback (most recent call last):
  Python Shell, prompt 14, line 1
builtins.TypeError: 'str' object is not callable
back_to_win.drive
'C:'
back_to_win.root
'\\'
wp.root
'\\'
back_to_win._parts
['C:\\', 'Program Files', 'Bazaar', 'crash.exe']
wp.parts
('C:\\', 'Program Files', 'Bazaar', 'crash.exe')
str(back_to_win)
'C:\\Program Files\\Bazaar\\crash.exe'
str(wp)
'C:\\Program Files\\Bazaar\\crash.exe'
`